README.md

```diff
@@ -1,257 +1,52 @@
 # YEAT
 
-YEAT, **Y**our **E**verday **A**ssembly **T**ool, is an update to [`asm_tools`](https://github.com/bioforensics/asm_tools). It uses a Snakemake workflow to preprocess paired-end, pacbio, and nanopore fastq reads and assemble them with various assembly algorithms.
+YEAT (**Y**our **E**verday **A**ssembly **T**ool) is an all-in-one platform designed to assemble multiple samples of varying read types (both short and long) using a combination of reliable, widely used, and cutting-edge algorithms such as SPAdes and Flye. It utilizes a Snakemake workflow to preprocess sample reads, perform assembly, and postprocess the resulting contigs in parallel.
 
-## Installation
+| Read Types  | Supported Algorithms |
+| ------------- | ------------- |
+| Paired-end  | SPAdes<sup>[1](#reference-1)</sup>, MEGAHIT<sup>[2](#reference-2)</sup>, Unicycler<sup>[3](#reference-3)</sup>, PenguiN<sup>[4](#reference-4)</sup>, VelvetOptimiser<sup>[5](#reference-5)</sup> |
+| Single-end | SPAdes<sup>[1](#reference-1)</sup>, MEGAHIT<sup>[2](#reference-2)</sup>, Unicycler<sup>[3](#reference-3)</sup>, PenguiN<sup>[4](#reference-4)</sup>, VelvetOptimiser<sup>[5](#reference-5)</sup> |
+| PacBio CLR (<20% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| PacBio Corrected (<3% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| PacBio HiFi (<1% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Hifiasm<sup>[8](#reference-8)</sup>, Hifiasm-meta<sup>[9](#reference-9)</sup>, Unicycler<sup>[3](#reference-3)</sup>, metaMDBG<sup>[10](#reference-10)</sup> |
+| ONT Regular (<20% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| ONT Corrected (<3% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| ONT High-quality (<5% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+
+## Installation (for Linux only)
```

Comment:
Here, the entire installation instructions are described as for Linux only. In the Wiki, only the metaMDBG and velvet installs are listed as Linux-only. A few points to consider:

- I think the approach you've begun in the Wiki is probably better. Point out which installation instructions are platform-specific, and which will work across the board.
- Is velvet-optimiser really not compatible with macOS? Both the README and the Wiki imply this, but I don't think that's the case. If it's not, you need to be careful the documentation doesn't imply something that's not true.
- The second point here highlights a common concern: when code or documentation is duplicated, it can easily get out of sync. You have to be disciplined to remember (or use a checklist to be reminded) to do the same update in multiple places. You might consider keeping the README more minimal if you want to include its current contents on the wiki.

Reply:
Moved the table and installation instructions to wiki to reduce the challenges of updating multiple places. I made the decision to move the table to the wiki because new algorithms will be added to the table in the future.